burn-book/src/building-blocks/tensor.md

```diff
@@ -260,7 +264,6 @@ Those operations are only available for `Float` tensors.
 | `tensor.log1p()`                             | `tensor.log1p()`                   |
 | `tensor.matmul(other)`                       | `tensor.matmul(other)`             |
 | `tensor.one_hot(index, num_classes, device)` | N/A                                |
```

Comment:
Not introduced by this PR, but I just realized that this should be `Tensor::one_hot(index, num_classes, device)`.

Would be good to fix the API doc especially since I think we should rename your current addition to `tensor.one_hot(num_classes)` (see comments below).

Reply:
Oh yeah good point I'll change that~